docs/StandardLibrary.md

```diff
@@ -157,7 +155,6 @@ Saturday  = 6
 
 Time and date functions.  Each of these accept a `time` value and return an integer.
 From the SPEC:
-[IMPL#155](https://github.com/influxdata/flux/issues/155)
 - `second` - integer Second returns the second of the minute for the provided time in the range [0-59].
```

Comment:
The spec says these functions are supposed to be in the `time` package. Can you communicate with @nathanielc to figure out which package they're supposed to be in? If they're supposed to be in `time`, move them there. If they're supposed to be in `date`, let's modify the spec to show that.

Reply:
I discussed this with Nathaniel. A `time` package was causing errors because it the name was already assigned to the `time` function, which is why we changed the package name to `calendar` (now `date`). I'll update the spec